cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,54 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+// Tailscale 1.14 and before stored state under %LocalAppData%
+// (usually "C:\WINDOWS\system32\config\systemprofile\AppData\Local"
+// when tailscaled.exe is running as a non-user system service).
+// However it is frequently cleared for almost any reason: Windows
+// updates, System Restore, even various System Cleaner utilities.
+//
+// We carefully move any state in AppData\Local to ProgramData
```

Comment:
Let's reword this from "we" to the subject (the func name) and make it first like https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences 

Like:

```
// tryWindowsAppDataMigration attempts to move the Windows state storage directory
// from its old location to the new location. (Issue NNN)
//
// Tailscale 1.14 and before ...
//
// This function does not return an error; on rename failure, code elsewhere
// in Tailscale will continue to use the old location as a fallback.

```

Reply:
Done.